fda-model/compute/simulation.cs

```diff
@@ -342,6 +351,40 @@ public static SimulationBuilder builder()
             return new SimulationBuilder(new Simulation());
         }
 
+        private bool LeveeIsValid()
```

Comment:
we probably want to implement framework validation on the model like i just did with statistics.